deltalake/sql/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "deltalake-sql"
+version = "0.1.0"
+edition = "2021"
```

Comment:
Before release this should pull version and license information from the workspace too